todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/util/schedulers/ImmediateSchedulerProvider.java

```diff
@@ -13,18 +13,18 @@
     @NonNull
     @Override
     public Scheduler computation() {
-        return Schedulers.immediate();
+        return Schedulers.trampoline();
```

Comment:
cool! I've been wanting to do this for a while!

Reply:
Does it deserve a comment explaining what it does?

Reply:
Added in the class Javadoc